index.html

```diff
@@ -1583,23 +1641,51 @@ <h5 id="Arcane Traits">Arcane Traits</h5>
                <td>Long Range Teleport
                </td>
                <td>
-                  <span class="wip">
-                     Maybe worth adding for other campaigns but I won’t use it for mine. 10
-                     minutes to cast a multi-person teleport to a known location within X miles.
-                  </span>
+                  This is an optional ability that is not used by default in all campaigns. It 
+                  is not a part of the Arcus lore but GM's may still wish to use it for other 
+                  campaigns.
+                  <p>
+                  You may spend an hour creating a magical symbol on the ground that is 10ft in diameter. 
+                  On completion this opens a gateway to another location that you have visited before. 
+                  the gateway persists for 6 seconds and any number of creatures may pass through it in 
+                  either direction - as long as they can fit through the 10ft gateway. 
+                  </p>
                </td>
                <td>1 Use
                </td>
                <td>Arcane Level 10
                </td>
             </tr>
+            <tr>
+               <td>Greater Dispel Magic
+               </td>
+               <td>
+                  This is an optional ability that is not used by default in all campaigns. It 
+                  is not a part of the Arcus lore but GM's may still wish to use it for other 
+                  campaigns.
+                  <p>
+                  You may dispel the magic properties from a greater magical item or area of stronger magic. 
+                  This might include most magical scroll or potion, a more complex weapon enchantment, 
+                  most magical traps or runes, or most areas cursed or empowered by a stronger source of magic. 
+                  The specific capabilities of this ability will be up to the GM's discretion.
+                  </p>
+               </td>
+               <td>3 AP - 3 Uses
+               </td>
+               <td>Arcane Level 10
+               </td>
+            </tr>
             <tr>
                <td>Phase Shift
                </td>
                <td>
-                  <span class="wip">
-                     Pass through walls for a short while, continue like you can with flight?
-                  </span>
+                  You may grant yourself 25ft of special movement for 2AP which can be used to pass through walls, 
+                  creatures, and other solid objects without harm - you are also immune to attacks of 
+                  opportunity while using this movement. This movement speed is
+                  separate from your other movement speed. You may recast this for 2AP each turn
```

Comment:
There is a little bit of while not usable in combat, a dude with this ability can just stay in phase shift for literally ever and walk through essentially anything, even mountains.  This feels like an ability which can break a lot of RP activities (any sort of jail break, maze, dungeon, trap, castle).  

I don't think this should be allowed to be recast without reusing the ability as it's already very strong without that